src/handlers/shared/start.ts

```diff
@@ -36,18 +36,12 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
     logger.error("Error while getting commit hash", { error: e as Error });
   }
 
-  // check max assigned issues
-  const openedPullRequests = await getAvailableOpenedPullRequests(context, sender.login);
-  logger.info(`Opened Pull Requests with approved reviews or with no reviews but over 24 hours have passed`, {
-    openedPullRequests: openedPullRequests.map((pr) => pr.html_url),
-  });
-
   const assignedIssues = await getAssignedIssues(context, sender.login);
```

Comment:
I'm pretty sure this can be removed as well as this is part of the checks function too